Fix for usage from China

emilpalsson opened the pull request:
Use China specific API URL if setting bootstrapURLKeys.region to 'cn'.

Fixes #493

itsmichaeldiego commented:
@emilpalsson Thanks for this, it is looking good, would you mind adding this to the docs? As soon as you do that I will release a new version fixing this.

jooj123 commented on src/utils/loaders/google_map_loader.js:
```diff
@@ -62,8 +62,12 @@ export default function googleMapLoader(bootstrapURLKeys) {
       ''
     );
 
+    const url = bootstrapURLKeys.region === 'cn'
```
what about other regions ?

itsmichaeldiego replied:
@jooj123 I believe the only region that has a problem with this is China. But we could confirm

emilpalsson replied:
Exactly, China is the only region mentioned on the Google Maps FAQ (see link in the issue), and the only region I've experienced problems with.

emilpalsson commented:
@itsmichaeldiego Great! Of course, I'll see if I can find a good spot for this in the docs right now.

emilpalsson commented:
@itsmichaeldiego I've added this to the docs, let me know if you want me to structure it differently. I also added a toLocaleLowerCase() for case insensitivity.

romanenko commented:
Looks like that if region is not set — it leads to uncaught Error now, with 0.28.0 release

![image](https://user-images.githubusercontent.com/336141/35142642-0f234b04-fd29-11e7-8e1b-f58f57330bc6.png)

I'm setting up my GoogleMap component like this:

```js
      <GoogleMap
        bootstrapURLKeys={{
          key: GOOGLE_MAPS_API_KEY,
          language: intl.locale,
        }}
//...
```

emilpalsson commented:
@romanenko Yes, I'm sorry for this. There is already a new PR to fix this (#494).
